soc/v2/nios2-qemu/Kconfig.soc

```diff
@@ -1,8 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_NIOS2_QEMU
-	bool "Nios II - Experimental QEMU emulation"
+	bool
 	select NIOS2
 	select HAS_MUL_INSTRUCTION
 	select HAS_DIV_INSTRUCTION
 	select HAS_MULX_INSTRUCTION
```

Comment:
such select's are Zephyr specifics and should be in `Kconfig`, the re-usable parts of the Kconfig soc tree should stay in `Kconfig.soc`.

Reply:
Fixed